blocks/magazine-footer/magazine-footer.css

```diff
@@ -0,0 +1,68 @@
+.section.magazine-footer-container {
+  background-color: #f8f8f8;
+}
+
+.section div.magazine-footer-wrapper {
+  max-width: 1170px;
+  align-items: center;
+}
+
+.magazine-footer {
+  background-color: #f8f8f8;
+  display: block;
+}
+
+.magazine-footer > div {
+  width: 100%;
+}
+
+.magazine-footer div.section-title {
+  text-transform: uppercase;
+  padding: 10px 0;
+  font-weight: 500;
+}
+
+.magazine-footer .col2 > *,
+.magazine-footer .col3 > * {
+  display: block;
+  padding: 6px 0;
+  line-height: 1.2em;
+  letter-spacing: 0.02em;
+}
+
+.magazine-footer h6.red {
+  color: #ee3124;
+}
+
+.magazine-footer picture {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 500px;
+}
+
+.magazine-footer img {
+  transform: rotate(-6deg);
+  box-shadow: -5px 5px 10px rgba(0 0 0 / 30%);
+  height: auto;
+}
+
+@media (min-width: 900px) {
+  .magazine-footer {
+    display: flex;
+    grid-template-columns: repeat(3, 1fr);
```

Comment:
grid template columns will not work here
 ![image](https://github.com/hlxsites/24life/assets/26192212/4dbce949-f164-4095-a948-d14efecf0598)